src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.BatchAI</PackageId>
 		<Description>Provides management capabilities for Azure Batch AI.</Description>
-		<Version>1.0.0-preview</Version>
```

Comment:
Is this meant to be a GA package? If yes, please update the `PackageReleaseNotes`

Reply:
PackageReleaseNotes have been updated